### ISSUE TITLE
RPi2 Jessie Test Segfaults after TestI18NWords2

### ISSUE THREAD

**Issue by hellbender3069** [libavg/libavg#744]
I have used a clean image of Jessie on the RPi2. Native compiled using parts of the x-compile instructions on the blog. All build ok, except some deprecated messages.

While running python Test.py the script segfaults after testI18NWords2

```
testDynamicWords (WordsTest.WordsTestCase) 
  testDynamicWords1: Difference image has avg=0.59, std dev=9.50

  testDynamicWords2: Difference image has avg=0.71, std dev=10.68
ok
testI18NWords (WordsTest.WordsTestCase) ... 
  testI18NWords1: Difference image has avg=8.78, std dev=40.30

  testI18NWords2: Difference image has avg=11.55, std dev=44.86
Segmentation fault
```

The memory split has been 128mb and 256mb, both of which result in the same issue.

**Comment by uzadow** [comment]
Interesting.
Can you get a stack trace (i.e. by starting the test in gdb?)

**Comment by uzadow** [comment]
Wait - when was your last pull of the jessie branch? This looks like something I fixed with 027a4e3b50e9a360a261f97d1e3080c618058ba3.

**Comment by hellbender3069** [comment]
Pulled new code from branch feature/raspi-jessie. Fixed the issue.
